### DISCUSSION TITLE
Developer meeting: 12-12-2022, 9am-11am

### DISCUSSION THREAD

**Discussion by hverhelst** [gismo/gismo#551] [Announcements]
# Developer meeting: 12-12-2022, 9am-11am

**Call for suggestions**

- @osmihran[#510]: The submodule mechanism: current status, what remains to be done and how can we start building our packages. 
- @hverhelst[#510]: update according to #510 all READMEs of the existing submodules
- @hverhelst[#510]: Adding unittests and add to PR testing framework
- @hverhelst: handling issues

**Comment by ckarampa** [comment]
One topic I would like to add is the proposal of a new way of creating Paraview files when using expressions ( and maybe other classes like gsField etc. in the future ). The proposed system facilitates the export of multiple fields (e.g. stresses and strains), multiple timesteps and export can be made at any path. This is inspired by what is proposed in #281, which is now 4 years old.  If anyone is interested, you can check it out on the branch: https://github.com/gismo/gismo/tree/paraview_multi . There is a file `examples/paraviewTest.cpp` that showcases the workflow.
